Gemfile

```diff
@@ -16,4 +16,5 @@ platforms :ruby do
   gem 'sqlite3'
   gem 'mysql2'
   gem 'pg'
+  gem 'mutex_m'
```

Comment:
Why?

Reply:
It looks as though I could move this into just the 6.0 and 6.1 variants of the gemfile. Not sure if that's the right way or not.